src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -26,5 +27,187 @@ public partial class AppDomain
                 AppContext.UnhandledException -= value;  
             }  
         }  
+
+        public string DynamicDirectory => null;
+        public string FriendlyName
+        {
+            get
+            {
+                Assembly assembly = Assembly.GetEntryAssembly();
+                string name = null;
+                if(assembly == null)
+                    name = "DefaultDomain";
+                else
+                    name = assembly.GetName().Name;
+                return name;             
+            }
+        }
+        public int Id => 1;
+        public bool IsFullyTrusted => true;
+        public bool IsHomogenous => true;
+        public event System.EventHandler DomainUnload { add { } remove { } }
```

Comment:
I added that because i was getting below error
error CS0067: The event 'AppDomain.DomainUnload' is never used

Adding unnecessary fluff silenced the compiler